examples/cli/readme.md

```diff
@@ -1 +1,619 @@
+
+```
+Models discovered during profiling:
+
+hello_world.py:
+	pytorch_model (executed 1x)
+		Model Type:	Pytorch (torch.nn.Module)
+		Class:		SmallModel (<class 'hello_world.SmallModel'>)
+		Location:	/home/jfowers/mlagility/examples/cli/hello_world.py, line 29
+		Parameters:	55 (<0.1 MB)
+		Hash:		479b1332
+		Status:		Model successfully benchmarked on NVIDIA A100-SXM4-40GB
+				Mean Latency:	0.027	milliseconds (ms)
+				Throughput:	21920.5	inferences per second (IPS)
+
+pytorch_outputs: tensor([-0.1675,  0.1548, -0.1627,  0.0067,  0.3353], grad_fn=<AddBackward0>)
+
+Woohoo! The 'benchmark' command is complete. Use the 'report' command to get a .csv file that summarizes results across all builds in the cache.
+```
+
+You can see that the device mentioned in the status is a `NVIDIA A100-SXM4-40GB`.
+
+## Multiple Models per Script
+
+The MLAgility tools will benchmark all models discovered in the input script. We can demonstrate this with the `two_models.py` script.
+
+Run the following command:
+
+```
+benchit scripts/two_models.py
+```
+
+To get a result like:
+
+```
+Models discovered during profiling:
+
+two_models.py:
+        pytorch_model (executed 1x)
+                Model Type:     Pytorch (torch.nn.Module)
+                Class:          SmallModel (<class 'two_models.SmallModel'>)
+                Location:       /home/jfowers/mlagility/examples/cli/scripts/two_models.py, line 32
+                Parameters:     55 (<0.1 MB)
+                Hash:           479b1332
+                Status:         Model successfully benchmarked on Intel(R) Xeon(R) CPU @ 2.20GHz
+                                Mean Latency:   0.000   milliseconds (ms)
+                                Throughput:     640717.1        inferences per second (IPS)
+
+        another_pytorch_model (executed 1x)
+                Model Type:     Pytorch (torch.nn.Module)
+                Class:          SmallModel (<class 'two_models.SmallModel'>)
+                Location:       /home/jfowers/mlagility/examples/cli/scripts/two_models.py, line 40
+                Parameters:     510 (<0.1 MB)
+                Hash:           215ca1e3
+                Status:         Model successfully benchmarked on Intel(R) Xeon(R) CPU @ 2.20GHz
+                                Mean Latency:   0.000   milliseconds (ms)
+                                Throughput:     642021.1        inferences per second (IPS)
+
+pytorch_outputs: tensor([ 0.3628,  0.0489,  0.2952,  0.0021, -0.0161], grad_fn=<AddBackward0>)
+more_pytorch_outputs: tensor([-0.1198, -0.5344, -0.1920, -0.1565,  0.2279,  0.6915,  0.8540, -0.2481,
+         0.0616, -0.4501], grad_fn=<AddBackward0>)
+
+Woohoo! The 'benchmark' command is complete. Use the 'report' command to get a .csv file that summarizes results across all builds in the cache.
+```
+
+You can see that both model instances in `two_models.py`, `pytorch_model` and `another_pytorch_model`, are both discovered and benchmarked.
+
+## Model Hashes
```

Comment:
Maybe introduce the concept of `--analyze-only` before this? You can't really know which hashes are available before analyzing the script.

Reply:
The hashes are right there in all the proceeding tutorials tho 👀

Reply:
Hmmmm. I see how this could be tricky. Maybe add a reference to `--analyze-only` here then?

Reply:
The new organization makes this much better! `--analyze-only` is now the first tutorial in `discovery.md`.

Reply:
Nice!